content/collections/guides_and_surveys/en/custom-css.md

```diff
@@ -0,0 +1,179 @@
+Guides and Surveys should match your brand and feel like part of your product. While [Themes](/docs/guides-and-surveys/themes) provide broad control over appearance, Custom CSS gives you fine-grained control over specific elements using CSS class selectors.
+
+{{partial:admonition type="note" heading="Web SDK only"}}
+Custom CSS is only available for the web SDK. Mobile SDKs (iOS, Android, React Native) don't support custom CSS.
+{{/partial:admonition}}
+
+## Overview
```

Comment:
Can we add a `(Beta)` suffix to the first heading? just so people have one more reason not to choose this over theme editing and we have an excuse if things change around a bit during the next couple weeks